pages/learn/welcome/introduction.md

```diff
@@ -8,17 +8,17 @@ title: Welcome
 
 Our [{{ $names.company.lower }} primer][primer] covers the platform from end to end, walking through all the pieces and how they fit together. We've also defined some [core concepts][concepts] you'll see throughout the documentation.
 
-Ready to dive in? The [getting started guides][getting-started] take you step-by-step through provisioning your first device and deploying your first application.
+Ready to dive in? The [getting started guides][getting-started] take you step-by-step through provisioning your first device and deploying your first fleet.
 
-Once you're up on the basics, we've got all the info you need to build even the most advanced applications and manage larger fleets:
+Once you're up on the basics, we've got all the info you need to build and manage advanced larger fleets:
 
 ### [Develop][develop]
 
-As you develop your application, these guides will help you configure and package your services, ensuring you have the proper container setup and that you can interact with hardware and system resources.
+As you develop your fleet, these guides will help you configure and package your services, ensuring you have the proper container setup and that you can interact with hardware and system resources.
```

Comment:
```suggestion
As you develop your application and grow your fleet, these guides will help you configure and package your services, ensuring you have the proper container setup and that you can interact with hardware and system resources.
```

I think this is one of those ones where it makes sense to keep it

Reply:
Agreed